src/Log/logSlice.ts

```diff
@@ -48,10 +52,16 @@ const slice = createSlice({
         toggleAutoScroll: state => {
             state.autoScroll = !state.autoScroll;
         },
+        toggleExtendedLogging: state => {
+            state.extendedLogging = !state.extendedLogging;
+            if (state.extendedLogging)
+                setLogLevel(getDeviceLibContext(), 'NRFDL_LOG_TRACE');
+            else setLogLevel(getDeviceLibContext(), 'NRFDL_LOG_CRITICAL');
```

Comment:
It is strange that the first `setLogLevel` is on a line by itself, while the other is directly after the `else`. Please add a line break after `else`.

Please also extract this into a function and call that also at https://github.com/NordicSemiconductor/pc-nrfconnect-shared/blob/b94bc284299c0c76dee885295d58b39f9011f52f/src/Log/logListener.ts#L74, so we only have the information, that `NRFDL_LOG_CRITICAL` is the default log level in one place.

Reply:
Fixed